MuhanguziDavid/url-shortener
# Setup docker and makefile

### What does this PR do?
- This PR sets up docker to run the app in a containerized environment
- adds commands to the Makefile to make it easy to run the application
- adds developer notes to notes.txt to guide other developers and users.

### How should I test this PR?
- Clone the repo
- In your terminal, cd into the project directory
- Run `make setup` to install dependencies
- Run `make server` to run the application in a containerized environment (docker)
- Open `http://192.168.99.100:8080/` or `http://localhost:8080/` in your browser. You should see the STORD URL Shortener running.
- After using the application, run `make stop` to stop the application.
- Run `make test` to run tests.
